templates/Register.html

```diff
@@ -23,22 +26,114 @@
         <input name="spotifytoken" id="spotify" type="checkbox">
     </p>
 
+
     <Label for="pin">4 character pin: </Label>
     <input name="pin" type="text" Placeholder="****" required pattern=".{4,4}" maxlength="4" minlength="4">
     <br>
     <p>Picture of face:</p>
-    <input name="facepath"  type="file" accept="image/*" capture="camera" required>
+    <input name="facepath" id="facepath"  type="file" accept="image/*" capture="camera" required>
 
     <hr/>
-
-    <input class="btn btn-lg btn-success" type="submit">
+    <!--can't get to post the way I want currently
+    "time();calendar(); spotifytoken(); googletoken(); maps(); pin(); weather();twittertoken(); name();email(); facepath(); postIt(); printC()"-->
+    <input onclick= "test()" class="btn btn-lg btn-success" type="submit">
     <input class ="btn btn-lg btn-danger" type="button" onclick="window.location.href='{{url_for('index')}}'" value="Cancel" />
 </form>
 
 <form action="{{url_for('logout')}}">
 
 </form>
 </body>
+<!--not sure how to call token-->
```

Comment:
what do you ben how to call token?

Reply:
I'm not sure how to get information once they login with the google account. The post is supposed to contain the googletoken, email, and username. I haven't figured out how to get that information yet so I can store it in a js object that I can then post with all the other info.

Reply:
I think I just don't understand how logging on with the google login works.  That method may work, we may need to talk about it later.

Reply:
either that or see if you can call {{currentUser}} as it is a variable that exists currenlty